openedx/core/lib/js_utils.py

```diff
@@ -68,3 +52,32 @@ def escape_json_dumps(obj, cls=EdxJSONEncoder):
     encoded_json = json.dumps(obj, ensure_ascii=True, cls=cls)
     encoded_json = _escape_json_for_html(encoded_json)
     return encoded_json
+
+
+def escape_js_str(js_str):
```

Comment:
I vote for #2-- esacpe_js_string.

I do tend to avoid abbreviations, but in this case, I like that the name of the method mirrors escapejs. In addition, I think "js" is a common enough term (and escape_javascript_string seems too long).

My second choice is "escape_javascript".